android/app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#FFFFFF</color>
```

Comment:
just guessing this is the splash screen during launch? could change to `#171717` instead of white

Reply:
didn't affect anything, i don't care if it's just white for now.